### PR TITLE
lifx-notifier 1.0.0

### PR DESCRIPTION
**Pull request by vgaidarji**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by vgaidarji** [comment]
I did a check in test bitrise workflow
```
workflows:
  primary:
    steps:
    - git::https://github.com/vgaidarji/steps-lifx-notifier.git@master:
        inputs:
        - auth_token: MY_TOKEN
        - bulb_label: MY_BULB_LABEL
        - workflow_start: 'yes'
        - color_build_started: 0000ff
        - color_build_success: 00ff00
        - color_build_failure: ff0000
    - activate-ssh-key@3.1.1:
        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
    - git-clone@3.4.0: {}
    - deploy-to-bitrise-io@1.2.5: {}
    - git::https://github.com/vgaidarji/steps-lifx-notifier.git@master:
        inputs:
        - auth_token: MY_TOKEN
        - bulb_label: MY_BULB_LABEL
        - workflow_start: 'no'
        - color_build_failure_custom: ‘000000’
```
and everything worked

screen shot 2016-10-08 at 5 21 38 pm

Also, I've tested step locally using `bitrise run test` workflow with temporary parameters set in `step.sh` (required parameters from `step.yml`).

File: steps/lifx-notifier/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      which requires OAuth2 Access token (https://api.developer.lifx.com/docs/authentication).
+    is_required: true
+    title: Authentication token
+- bulb_label: null
+  opts:
+    description: |
+      This is the name of the LIFX bulb, eg. "Bedroom", "Office".
+    is_required: true
+    title: LIFX bulb label
+- opts:
```
Comment (viktorbenei) [inline steps/lifx-notifier/1.0.0/step.yml]: why is this required? couldn't you just simply add two steps, one at start where you specify the color "yellow" for example, and then another one at the end with green/red? Just asking, it simply seems to be more complex than what would be required IMO :)

File: steps/lifx-notifier/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+published_at: 2016-10-08T18:06:20.182318893+03:00
+source:
+  git: https://github.com/vgaidarji/steps-lifx-notifier.git
+  commit: 52ea2c3561022d0664e6f6af1ce7460a14009698
+host_os_tags:
+- osx-10.8
+- osx-10.10
+- ubuntu
+project_type_tags:
+- ios
```
Comment (viktorbenei) [inline steps/lifx-notifier/1.0.0/step.yml]: no need to add any project type tag if it works with any kind of project

File: steps/lifx-notifier/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+- android
+type_tags:
+- utilities
```
Comment (viktorbenei) [inline steps/lifx-notifier/1.0.0/step.yml]: maybe also `notification`? e.g. the Slack step has that

**Comment by viktorbenei** [comment]
Thank you so much for sharing this step!

I left a couple of notes, which could improve the step, but there was no "show stopper" issue, so these things can be done as part of a step update, if you have the time, but the first version is now available! ;)

**Comment by viktorbenei** [comment]
Actually there's an issue with another step in the steplib, which we'll try to sort out ASAP - until that the step won't show up on the web UI unfortunately.. But this should be resolved ASAP

File: steps/lifx-notifier/1.0.0/step.yml (hunk 1 above)
Comment (vgaidarji) [inline steps/lifx-notifier/1.0.0/step.yml]: How can I understand that the workflow started? If there's such an option in bitrise, than definitely we can improve the step.

@viktorbenei I can find in [available-environment-variables](http://devcenter.bitrise.io/docs/available-environment-variables) only `BITRISE_BUILD_STATUS` variable which indicates "success" or "failure". What's the value of this variable at the start of the workflow?

File: steps/lifx-notifier/1.0.0/step.yml (hunk 2 above)
Comment (vgaidarji) [inline steps/lifx-notifier/1.0.0/step.yml]: Issue created https://github.com/vgaidarji/steps-lifx-notifier/issues/1

File: steps/lifx-notifier/1.0.0/step.yml (hunk 3 above)
Comment (vgaidarji) [inline steps/lifx-notifier/1.0.0/step.yml]: Issue created https://github.com/vgaidarji/steps-lifx-notifier/issues/2

File: steps/lifx-notifier/1.0.0/step.yml (hunk 1 above)
Comment (vgaidarji) [inline steps/lifx-notifier/1.0.0/step.yml]: Thanks for the tip. Issue created https://github.com/vgaidarji/steps-lifx-notifier/issues/3
